flink-connectors/flink-connector-kafka/pom.xml

```diff
@@ -111,11 +111,12 @@ under the License.
 		</dependency>
 
 		<!-- Required to execute the kafka server for testing. Please change the zookeeper version accordingly when changing the Kafka version
+			 Currently synced with Kafka 3.2.1
              https://github.com/apache/kafka/blob/839b886f9b732b151e1faeace7303c80641c08c4/gradle/dependencies.gradle#L122 -->
 		<dependency>
 			<groupId>org.apache.zookeeper</groupId>
 			<artifactId>zookeeper</artifactId>
-			<version>3.5.9</version>
+			<version>3.6.3</version>
```

Comment:
Ideally this sticks with `${zookeeper.version}` if possible.

Reply:
Done